### ISSUE TITLE
forwardMessages is not working on 4.35.13

### ISSUE THREAD

**Issue by honggianto** [open-wa/wa-automate-nodejs#2699]
### Are you using the latest version of the library?

- [X] I have checked and am using the latest version of the library.

### What type of session are you experiencing this issue on?

Multi-device and I have set multiDevice to true in my config OR am using the --multi-device flag

### What type of host account are you experiencing this issue on?

Personal account (normal)

### Mode

EASY API/CLI

### Steps To Reproduce

/forwardMessages stalled.

### DEBUG INFO

```markdown
{
  "WA_VERSION": "2.2216.6",
  "WA_AUTOMATE_VERSION": "4.35.13",
  "BROWSER_VERSION": "HeadlessChrome/99.0.4844.51",
  "START_TS": 1652151938730,
  "RAM_INFO": "Total: 1.56 GB | Free: 0.61 GB",
  "PPTR_VERSION": "UNKNOWN",
  "LATEST_VERSION": true,
  "CLI": true,
  "LAUNCH_TIME_MS": 30440,
  "ACC_TYPE": "PERSONAL",
  "chats": 41,
  "messages": 50,
  "contacts": 1123,
  "isMd": true,
  "INSTANCE_ID": "748a4896-1fbc-4de4-abce-ccae44e6870a"
}
```

### Environment

```markdown
- OS: Linux 5.4
- Node: 14.19.1
- npm: 8.8.0
```

### Screenshots/Logs

info: IN fe324  {"_args":{"messages":"false_120363040637821590@g.us_DD16B43112A876381988BBDE6C71DD4F_628175055558@c.us","to":"120363039685598965@g.us"},"_method":"forwardMessages","timestamp":"2022-05-10T03:15:59.225Z"}
info: Page Console: err JSHandle@object {"timestamp":"2022-05-10T03:15:59.248Z"}
info: Page Console: res JSHandle@array {"timestamp":"2022-05-10T03:16:03.495Z"}

**Comment by honggianto** [comment]
Trying version 4.36.0 and here's the verbose debug result 👍 

info: OUT d1924: 273ms  {"res":"true_120363039685598965@g.us_3EB053464AA6BDEA56F5_6282229751750@c.us","timestamp":"2022-05-12T14:32:48.985Z"}
info: IN 21af0  {"_args":{"messages":"false_120363040637821590@g.us_3EB0ABB5F61A0C69407C_628175055558@c.us","to":"120363039685598965@g.us"},"_method":"forwardMessages","timestamp":"2022-05-12T14:32:48.997Z"}
info: Page Console: err JSHandle@object {"timestamp":"2022-05-12T14:32:49.025Z"}
info: Page Console: Failed to load resource: net::ERR_FAILED {"timestamp":"2022-05-12T14:32:49.544Z"}
info: Page Console: res JSHandle@array {"timestamp":"2022-05-12T14:32:51.066Z"}

**Comment by honggianto** [comment]
Information Update : 

The forwardMessages was working on 4.35.13 and 4.36.0. But... there's a little bug I've found.
Here's the scenario that can trigger the bug : 

1. I have two Whatsapp Group (WAG). Let say WAG A and WAG B
2. When someone post a picture / video in the WAG A, I called the forwardMessages immediately, to forward the picture / video sent from WAG A to WAG B.
3. After the forwardMessages executed, my script loop back to call getUnreadMessages.
4. Here's the problem (little bug that I've found) It seems this WA Automation won't get any unread messages as long as no new messages in the WAG B.
5. Hopefully SHAH can fix this little bug 🙏
